charts/k8up/values.yaml

```
@@ -132,3 +132,13 @@ resources:
     cpu: 20m
     # -- Memory request of K8up operator. See [supported units][resource-units].
     memory: 128Mi
+
+cleanup:
+  # -- Cleanup-job image pull policy
+  pullPolicy: IfNotPresent
+  # -- Cleanup-job image registry
+  registry: hub.docker.com
```

```suggestion
  registry: docker.io
```

The registry URL is wrong.